Automattic/wp-calypso
# Signup: Add new a/b test introducing the import flow on site type step

#### Changes proposed in this Pull Request

* Adds a new a/b test (`showImportFlowInSiteTypeStep`) to the site-type step of signup.
* Displays "Already have a website?" link beneath site type options.
* Clicking on new link takes you to the `import-onboarding` flow.
* `import-onboarding` is a duplicate of the `import` flow, but with the `site-type` step added to prevent issues when switching flows.
* All of this is behind the `signup/import-flow` feature flag, to unblock it before other import signup flow work is complete, and allow an easy off switch if it doesn't test well.

Before:
![image](https://user-images.githubusercontent.com/363749/61406646-e2d01700-a8a1-11e9-8bee-b88849c3634e.png)


After (with feature-flag enabled and ab-test value of `show`):
![image](https://user-images.githubusercontent.com/363749/61406567-b87e5980-a8a1-11e9-8faa-c987de327398.png)


#### Testing instructions

* Enable the feature-flag `signup/import-flow` if you're testing in an environment other than development.
* Add yourself to the `showImportFlowInSiteTypeStep` with the test value `show` (use the "dev" bug in the lower right-hand corner of the screen).
* Visit `/start` as an existing user. You should be redirected to `/start/site-type`. Verify that the "Already have a website?" link is displayed.
* Click the "Already have a website?" link. Verified that you are taken to the import flow.
* Leave the feature-flag enabled.
* Create a new jurassic.ninja site and note the URL.
* Visit `/jetpack/connect` and enter the site URL.
* Continue to the site type step and verify that "Already have a website?" is not visible.
* Disable the feature-flag.
* Visit `/start` as an existing user. You should be redirected to `/start/site-type`. Verify that the "Already have a website?" link is NOT displayed.
